src/App.jsx

```diff
@@ -1,12 +1,16 @@
 //  TODO APP : Designed to cheer for each little accomplishment we make as we progress toward our goals - like building an app. So this app cheers for you when you complete each task and celebrates with you when you complete all of your tasks.
 
+import { BrowserRouter, Routes, Route, Link } from 'react-router-dom';
+import { MdOutlinePlaylistAddCheck, MdArrowBackIosNew } from 'react-icons/md';
 import React, { useState, useEffect } from 'react';
+import { toast } from 'react-toastify';
+
 import AddTodoForm from './components/AddTodoForm';
 import TodoList from './components/TodoList';
 import CompletedTodos from './components/CompletedTodos';
-import { BrowserRouter, Routes, Route, Link } from 'react-router-dom';
-import { ToastContainer, toast } from 'react-toastify';
-import { MdOutlinePlaylistAddCheck, MdArrowBackIosNew } from 'react-icons/md';
+import { ConfiguredToast } from './components/ConfiguredToast';
+
+import 'react-toastify/dist/ReactToastify.css';
```

Comment:
I tend to organize imports by grouping them into packages and internal files. Each group then sorted alphabetically.